### PR TITLE
Arabidopsis training update: fix results interpretation

### PR DESCRIPTION
**Pull request by gallardoalba**
This PR corrects the interpretation of the results.

### CODE REVIEW

**Review by bgruening — DISMISSED** [review]

File: topics/transcriptomics/tutorials/mirna-target-finder/tutorial.md [hunk 1]
```diff
@@ -634,21 +635,16 @@ Now we continue with the DE genes analysis.
 > <question-title></question-title>
 >
 > 1. How many genes show statistically significant differential expression?
-> 2. How many of them are upregulated? And downregulated?
+> 2. How many of them are significally upregulated (at least two-fold change)? And downregulated?
```
Comment (shiltemann) [inline topics/transcriptomics/tutorials/mirna-target-finder/tutorial.md:638]:
```suggestion
> 2. How many of them are significantly upregulated (at least two-fold change)? And downregulated?
```

File: topics/transcriptomics/tutorials/mirna-target-finder/tutorial.md [hunk 2]
```diff
@@ -634,21 +635,16 @@ Now we continue with the DE genes analysis.
...
 > 3. What is the most significantly DE downregulated gene and what is it biological function?
 >
 > > <solution-title></solution-title>
 > >
 > > 1. The total number of genes whose expression differential between the two experimental conditions is 4176.
-> > 2. Of them, 328 are downregulated by the BR treatment and 778 are upregulated.
+> > 2. Of them, 328 are significally upregulated by the BR treatment and 778 are downregulated.
```
Comment (shiltemann) [inline topics/transcriptomics/tutorials/mirna-target-finder/tutorial.md:644]:
```suggestion
> > 2. Of them, 328 are significantly upregulated by the BR treatment and 778 are downregulated.
```

**Review by shiltemann — COMMENTED** [review]
Thanks @gallardoalba!

**Comment by shiltemann** [comment]
Thanks!

**Pull request merged**